frontend/src/views/camp/navigation/mobile/NavSidebar.vue

```diff
@@ -84,7 +84,7 @@ import UserAvatar from '@/components/user/UserAvatar.vue'
 import SidebarListItem from '@/components/layout/SidebarListItem.vue'
 
 export default {
-  name: 'NavMobileSidebar',
+  name: 'navSidebar',
```

Comment:
Genau so was wie diese ESLint-Regel.
```suggestion
  name: 'NavSidebar',
```